packages/wonder-blocks-icon-button/index.test.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import React from "react";
+import {mount} from "enzyme";
+
+import IconButton from "./index.js";
+
+describe("IconButton", () => {
```

Comment:
It's nice that all the heavy lifting can be done by `ClickableBehavior` and that it can be reused between components.